apps/finance/test/finance.js

```diff
@@ -176,9 +180,9 @@ contract('Finance App', ([root, owner, recipient]) => {
         await financeApp.mockIncreaseTime(100) // to make sure it overflows with MAX_UINT64 period length
         // initialize with MAX_UINT64 as period duration
         await financeApp.initialize(vault.address, MAX_UINT64)
-        const [isCurrent, start, end, firstTx, lastTx] = await financeApp.getPeriod(await financeApp.currentPeriodId())
+        const data = await financeApp.getPeriod(await financeApp.currentPeriodId())
 
-        assert.equal(end.valueOf(), MAX_UINT64.valueOf(), "should have set the period's end date to MAX_UINT64")
+        assert.equal(data.endTime.eq(MAX_UINT64), true, "should have set the period's end date to MAX_UINT64")
```

Comment:
We use BN `eq` function to compare two BN numbers

Reply:
Noted in Voting as well, but let's use the string serialization instead of `.eq()` to make the failure case more explicit in output logs :)